### ISSUE TITLE
base:pinned  ansible:latest rust:pinned  build-tools:pinned，Where can I download these images

### ISSUE THREAD

**Issue by william-lbn** [neondatabase/neon#7528]
369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned .    I can find it on Dockerhub .   neondatabase/rust:pinned
369495373322.dkr.ecr.eu-central-1.amazonaws.com/build-tools:pinned   I can find it on Dockerhub .   neondatabase/build-tools:pinned
369495373322.dkr.ecr.eu-central-1.amazonaws.com/base:pinned.    I can't find it on Dockerhub
369495373322.dkr.ecr.eu-central-1.amazonaws.com/ansible:latest.   I can't find it on Dockerhub

**Comment by william-lbn** [comment]
Hi @skyzh 

Run your GitHub Actions locally by act and  Found that mirroring is needed. base:pinned and ansible:latest， Is there an alternative image?

**Comment by skyzh** [comment]
I don't think so; and you do not need these two images in order to build neon.
